nextpvr2m3u.py

```diff
@@ -43,25 +43,18 @@ def GetListOfSubstrings(stringSubject,string1,string2):
                 continueloop=0
         return MyList
 
-    def load_url(url):
-        URL_Req = urllib2.Request(url)
-        
-        try:
-            URL_Response = urllib2.urlopen(URL_Req)
-            URL_Source = URL_Response.read()
-        except urllib2.HTTPError as e: #https://www.programcreek.com/python/example/68989/requests.HTTPError
-            URL_Source = ""
-        
-        URL_Source = URL_Source.replace('\t', '') #remove tabs
-        
-        return URL_Source
-    
-    def load_json(url):
-        req = urllib2.Request(url)
-        opener = urllib2.build_opener()
-        f = opener.open(req)
-        result = json.loads(f.read())
-        return result
+
+    def load_url(url: str):
```

Comment:
Please update this file based on latest in master (had to remove unnecessary imports and functions).